rdfservice.umd.js

```diff
@@ -0,0 +1 @@
faultNamespace(){return this.prefixDict[":"]}getPrefix(t){const e=Object.keys(this.prefixDict),r=Object.values(this.prefixDict);return e.find((s,a)=>r[a]===t)||t}shorten(t){const r=Object.keys(this.prefixDict).find(s=>t.includes(this.prefixDict[s]));return r?t.replace(this.prefixDict[r],r):t}getSparqlPrefix(t){if(t in this.prefixDict)return`PREFIX ${t} <${this.prefixDict[t]}> `;throw P+t}get sparqlPrefixes(){let t="";for(let e in this.prefixDict)t=t+`PREFIX ${e} <${this.prefixDict[e]}> `;return t}mintUri(t=this.defaultNamespace){return t+crypto.randomUUID()}async runQuery(t){if(d(t))throw Error("runQuery: A valid query is required");const e=await fetch(this.queryEndpoint+encodeURIComponent(this.sparqlPrefixes+t));if(!e.ok)throw e.statusText;return await e.json()}async runUpdate(t,e){let r=e??this.defaultSecurityLabel;d(r)&&console.warn("Security label is being set to an empty string. Please check your security policy as this may make the data inaccessible");var s={method:"POST",headers:{Accept:"*/*","Content-Type":"application/sparql-update"},body:this.sparqlPrefixes+t};const a=await fetch(this.updateEndpoint,s);if(!a.ok)throw a.statusText;return a.text()}async insertTriple(t,e,r,s,a,l){var f=E(this,p,v).call(this,r,s,l);return await this.runUpdate("INSERT DATA {<"+t+"> <"+e+"> "+f+" . }",a)}async deleteTriple(t,e,r,s,a){var l=E(this,p,v).call(this,r,s,a);return await this.runUpdate("DELETE DATA {<"+t+"> <"+e+"> "+l+" . }")}async deleteNode(t,e=!1){if(d(t))throw Error(o);await this.runUpdate("DELETE WHERE {<"+t+"> ?p ?o . }"),e||await this.runUpdate("DELETE WHERE {?s ?p <"+t+">}")}async deleteRelationships(t,e){if(d(t))throw Error(o);if(d(e))throw Error("Cannot have an empty predicate");await this.runUpdate(`DELETE WHERE {<${t}> <${e}> ?o . }`)}async instantiate(t,e,r){if(d(t))throw Error("Cannot have an empty cls");return e||(e=this.mintUri()),await this.insertTriple(e,this.rdfType,t,void 0,r),e}async getRelated(t,e){var f;if(d(t))throw new Error(o);if(d(e))throw new Error(h);const r=`SELECT ?related WHERE {<${t}> ?pred ?related . ?pred <${this.rdfsSubPropertyOf}>* <${e}> .}`,s=await this.runQuery(r);return(f=s==null?void 0:s.results)!=null&&f.bindings?s.results.bindings.map(w=>w.related.value):[]}async getRelating(t,e){var l;if(d(t))throw new Error(o);if(d(e))throw new Error(h);const r=`SELECT ?relating WHERE {?relating ?pred <${t}> . ?pred <${this.rdfsSubPropertyOf}>* <${e}> . }`,s=await this.runQuery(r);return(l=s==null?void 0:s.results)!=null&&l.bindings?s.results.bindings.map(f=>f.relating.value):[]}}p=new WeakSet,v=function(t,e="URI",r){if(e==="URI")var s=`<${t}>`;else if(e=="LITERAL"){var s=`"${t}"`;r&&(s=`${s}^^${r}`)}else throw new Error("unknown objectType");return s},n.RDFProperty=c,n.RDFSResource=u,n.default=x,n.emptyPredicateErrorMessage=h,n.emptyUriErrorMessage=o,n.noColonInPrefixException=m,n.unknownPrefixException=P,n.unrecognisedIdField=b,Object.defineProperties(n,{__esModule:{value:!0},[Symbol.toStringTag]:{value:"Module"}})});
```

Comment:
Perhaps `rdfservice.mjs` and `rdfservice.umd.js` and `types.d.ts` were result of experiementing with builds and can be deleted?

I would have expected these files in `./packages/rdfservice`